### PR TITLE
heads now get access to the captain's quarters on lowpop

### PR DESCRIPTION
**Pull request by Amelia0010**
### Intent of your Pull Request

p self explanatory

#### Changelog

:cl:  
tweak: heads now get access to the captain's quarters on lowpop
/:cl:

### CODE REVIEW

**Comment by monster860** [comment]
I can't wait for the lowpop CE purchasing a build-your-own-shuttle

**Comment by Amelia0010** [comment]
:^)

File: yogstation/code/modules/jobs/job_types/captain.dm [hunk 1]
```diff
@@ -1,3 +1,7 @@
 /datum/job/captain
 	exp_requirements = 300
-	exp_type_department = EXP_TYPE_COMMAND
\ No newline at end of file
+	exp_type_department = EXP_TYPE_COMMAND
+
+/datum/job/hop/New()
+	minimal_access += ACCESS_CAPTAIN
```
Comment (monster860) [inline yogstation/code/modules/jobs/job_types/captain.dm]: **EXPECTATION**
adding captain to heads during lowpop rounds
**REALITY**
adding captain to heads during highpop rounds

**Review by monster860 — CHANGES_REQUESTED** [review]

File: yogstation/code/modules/jobs/job_types/captain.dm (hunk 1 above)
Comment (Amelia0010) [reply yogstation/code/modules/jobs/job_types/captain.dm]: I hate the way everything works sometimes

**Comment by monster860** [comment]
I see you've got ULTIMIATE LAZYNESS using the web editor

**Comment by Amelia0010** [comment]
yep

**Comment by ThatLing** [comment]
This always gives them captain access?

**Comment by Amelia0010** [comment]
no, if it's highpop it uses minimal_access rather than access

**Comment by ThatLing** [comment]
WHAT COULD GO WRONG

**Pull request merged**